src/sidebar/components/menu-item.js

```diff
@@ -220,10 +220,10 @@ export default function MenuItem({
     <Fragment>
       {menuItem}
       {hasSubmenuVisible && (
-        <Slider visible={isSubmenuVisible}>
+        <Slider visible={/** @type {boolean} */ (isSubmenuVisible)}>
```

Comment:
I guess it can't see that this _has_ to be a **boolean** due to the `hasSubmenuVisible` check above it. I wonder if thats  a jsx issue with ts or something else...